lambda/kendra-webcrawler-schedule-updater/Makefile

```diff
@@ -1,6 +1,6 @@
 NAME=$(shell basename $(shell pwd))
 DST=../../build/lambda/$(NAME).zip
-RESOURCES := $(shell  find . | grep -v node_modules | grep -v test )
+RESOURCES := $(shell  find . | grep -v test )
 
 $(DST): $(RESOURCES) 
 	echo "Building $(NAME)"; pip3 install -r requirements.txt -t . && zip -r -q $(DST) .
```

Comment:
The Makefiles should, in theory, also install python modules to subdirectory 'py_modules' as we do elsewhere.
It's kind of moot since the requiremenst.txt is empty for them all, so we can defer this to later if you want.

Reply:
good point. I think we should take a look at all our makefiles and update. I did a scan for a few, but i think we should review this more.